cuda/base/cusparse_bindings.hpp

```diff
@@ -879,7 +879,7 @@ inline void destroy(cusparseSpMatDescr_t descr)
 }
 
 
-#if (CUDA_VERSION >= 11030)
+#if (CUDA_VERSION > 11030)
```

Comment:
nit: List the version it was introduced in, not the last one that didn't have it
```suggestion
#if (CUDA_VERSION >= 11031)
```

Reply:
Wasn't sure in which way I should do it, thanks! Now it is the proper way.